ec2/ec2.go

```diff
@@ -32,7 +32,7 @@ const (
 	debug = false
 
 	// apiVersion is the AWS API version used for all EC2 requests.
-	apiVersion = "2015-03-01"
+	apiVersion = "2016-11-15"
```

Comment:
That feels like we're still using a really really old version, is there newer stuff that we explicitly don't want to use?

Reply:
We were using an even older version before that but I had to bump it to get the public IPs in the response. This is yet another bump to the latest API version according to [this](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/EC2.html) which I dug from the official JS API (apparently, finding an actual _list_ of the API versions is a non-trivial task!).

I believe the API version was not bumped before that for fear of breaking stuff, perhaps due to API changes. However, since we (and SQA) pretty much thoroughly test ec2 deployments, bumping the API shouldn't be an issue.

I guess we should switch to the goamz package or the amazon one at some point since it is actively maintained (and ahead of our fork).